axum/src/body/mod.rs

```diff
@@ -22,12 +23,31 @@ pub use bytes::Bytes;
 pub type BoxBody = http_body::combinators::UnsyncBoxBody<Bytes, Error>;
 
 /// Convert a [`http_body::Body`] into a [`BoxBody`].
+///
+/// If the given body is already a [`BoxBody`], this function will not box the body again.
```

Comment:
Hm this feels like an implementation detail thats probably not worth documenting 🤔